server/src/internalClusterTest/java/org/elasticsearch/cluster/coordination/RareClusterStateIT.java

```diff
@@ -177,6 +178,7 @@ public void onFailure(Exception e) {
         return future;
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/88647")
```

Comment:
I believe this is outdated, test was fixed in https://github.com/elastic/elasticsearch/pull/91272

Reply:
Good point, addressed in 4e4a7b6aa14d5126fc562de6a1d25869aa53acbc.